api/testtriggers/v1/testtrigger_types.go

```diff
@@ -66,6 +68,8 @@ type TestTriggerSpec struct {
 	Delay *metav1.Duration `json:"delay,omitempty"`
 	// whether test trigger is disabled
 	Disabled bool `json:"disabled,omitempty"`
+	// Target helps decide on which runner the execution is scheduled.
+	Target *commonv1.Target `json:"target,omitempty" expr:"include"`
```

Comment:
I think it should go to `TestTriggerActionParameters`, as we have `config` and `tags` there.